lib/schema/organizationContact.ts

```diff
@@ -0,0 +1,11 @@
+import * as Joi from 'joi';
+import { Regexes } from '../regexes';
+
+export const OrganizationContactSchema = Joi.object().keys({
+  firstName: Joi.string().max(32).regex(Regexes.lettersAndSymbolsOnly).required(),
+  lastName: Joi.string().max(32).regex(Regexes.lettersAndSymbolsOnly).required(),
+  email: Joi.string().max(100).regex(Regexes.email).required(),
+  phone: Joi.string().length(10).regex(Regexes.numbersOnly).allow(null).empty(''),
+  phoneExtension: Joi.string().max(6).allow(null).empty(''),
+  id: Joi.any()
```

Comment:
@peteplays @jrmce  Maybe I'm wrong, but I think we should only be allowing fields that can be updated correct? I don't think the `id`'s are updatable via an api request. Does this sound right?

Reply:
Agreed - @peteplays @seescode stuff like `id`, `updated_at` and `created_at` should not live in these schemas - unless we are marking them explicitly as NOT allowed.